.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "all",
```

Comment:
Are you okay with trailing commas in function call?

Reply:
Yes. It's never been an issue :)